org.eclipse.xtext.xtext.ui.examples/projects/domainmodel/org.eclipse.xtext.example.domainmodel.ui/src/org/eclipse/xtext/example/domainmodel/ui/navigation/DomainmodelHyperlinkHelper.java

```diff
@@ -84,6 +91,16 @@ public void createHyperlinksByOffset(XtextResource resource, int offset, IHyperl
 				}
 			}
 		}
+		
+		createHyperlinksInJavadoc(resource, offset, acceptor);
 	}
 
+	private void createHyperlinksInJavadoc(XtextResource resource, int offset, IHyperlinkAcceptor acceptor) {
```

Comment:
i wonder if we should make this available via injectable boolean in xbase
@szarnekow what do you think?

Reply:
Or we make this available by default and opt-out would be to override the method?

Reply:
:+1:

Reply:
Actually I wonder why the `DomainmodelHyperlinkHelper` inherits from `TypeAwareHyperlinkHelper` and not from `XbaseHyperLinkHelper`. The `XbaseHyperLinkHelper` class is already aware of [createHyperlinksInJavaDoc](https://github.com/eclipse/xtext-eclipse/blob/2027a7e48acafb161bc3ee2d861a3b3e36cc8d6d/org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/navigation/XbaseHyperLinkHelper.java#L162).

Reply:
XbaseHyperLinkHelper seems way newer than TypeAwareHyperlinkHelper. thus this seens to be a migration issue when XbaseHyperLinkHelper was introduced

Reply:
Then simply changing the inheritance would sufficient, isnt it?

Reply:
Assume so

Reply:
@szarnekow What do you think?

Reply:
ping @szarnekow

Reply:
True. It certainly looks like that would be the right thing to do.